02-Git/rps.c

```diff
@@ -0,0 +1,7 @@
+#include "stdio.h"
+
+int main(void)
+{
+       return 1;
```

Comment:
why not zero == success?

Reply:
Fixed.
PS: Technically you've said that will not check code itself, only commit structure.